scss/dialogs/_delta-dialog.scss

```diff
@@ -0,0 +1,26 @@
+.delta-dialog-card__inner-without-padding {
+  margin: -20px;
+}
+
+.delta-dialog-card--no-padding {
+  padding: 0px;
+}
+
+.delta-dialog-card {
+  background-color: var(--bp3DialogHeaderBg); 
+  padding: 20px;
+  height: 100%;
+  &:last-child {
+    height: -webkit-fill-available;
+  }
+}
+
+.bp3-small-dialog {
```

Comment:
this looks like bp3 overwrite and it isn't scoped so I suggest moving this class to the bp3 overwrites file.

Reply:
It's not an overwrite, just bad naming.